graylog2-web-interface/src/stores/streams/StreamsStore.ts

```diff
@@ -137,6 +137,10 @@ class StreamsStore {
   sendDummyAlert(streamId: string) {
     const url = URLUtils.qualifyUrl(ApiRoutes.StreamAlertsApiController.sendDummyAlert(streamId).url);
     const promise = fetch('POST', url);
+    promise.then(
+      () => UserNotification.success('Test notification was sent successfully'),
+      (error) => UserNotification.error('Could not send test notification')
```

Comment:
That's a good point. I thought that would raise an exception when sending the test email, but I didn't test. I want to change the test to work on a single notification instead of on all notifications of a stream, and can look into that then.